ui/src/data/containers/DetailPageContainer/__tests__/DetailPageContainer.test.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { render } from '@testing-library/react';
+import { fromJS } from 'immutable';
+import { Provider } from 'react-redux';
+
+import DetailPageContainer from '../DetailPageContainer';
+import { getStoreWithState } from '../../../../fixtures/store';
+
+describe('DetailPageContainer', () => {
+  it('renders initial state', () => {
+    const store = getStoreWithState({
```

Comment:
Better use getStore instead of getStoreWithState. getStoreWithState should be removed